### PR TITLE
Add UART1 whitelist to TCP server/client and UDP client [master]

### PR DESCRIPTION
**Pull request by cbeighley**
For https://github.com/swift-nav/piksi_v3_bug_tracking/issues/992.

### CODE REVIEW

File: package/piksi_system_daemon/piksi_system_daemon/src/whitelists.c [hunk 1]
```diff
@@ -56,68 +56,226 @@ static port_whitelist_config_t port_whitelist_config[PORT_MAX] = {
...
+        MsgBasePosECEF                72
+        MsgObs                        74
+        MsgGloBiases                 117        
+        MsgHeartbeat               65535
+    */
+
   },
   [PORT_UART1] = {
     .name = "uart1",
-    .wl = "23,29,65,72,74,80,117,134,136,144,163,165,166,167,171,175,181,185,187,188,257,258,259,520,522,524,526,527,528,1025,2304,2305,2306,65280,65282,65535"
```
Comment (silverjam) [inline package/piksi_system_daemon/piksi_system_daemon/src/whitelists.c]: 175 is being dropped by this change, which is needed for the recent settings write response changes

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018012001-10-g7133a73
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018012001-10-g7133a73
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-10-g7133a73

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018012001-14-gc48ce22
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-14-gc48ce22/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018012001-14-gc48ce22/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018012001-14-gc48ce22

**Pull request merged**